projects/module-1/colorandom.md

```diff
@@ -124,113 +121,122 @@ Here's a list of possible extensions to implement - but **ONLY IF** your team ha
+- PRs have clear, thorough descriptions and are consistently reviewed by other team members before work has been merged.
+- README is formatted and gives new contributors and employers sufficient context about the project including setup instructions, deploy link (gh-pages), images or video of the functioning app, technologies used, reflections, future features, and other pertinent information.
+
+✨WOW✨ can look like:
+- The team holds regular standups and retros while also revisiting the norming document as better perspective is gained
+- A [PR template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) is effectively used
+</section>
+
+<section class="answer">
+### Comp Recreation
+
+- Application implements all major comp details accurately and correctly on desktop (layout, spacing, alignment, etc)
+- Careful attention was given to the little details like hover states.
+
+✨WOW✨ can look like:
+- Additional elements and animations have been added that match the visuals established in the comps.
+- The design is responsive across multiple screen sizes including tablets and mobile devices.
+</section>
+
+<section class="answer">
+### HTML & CSS
+
+- Crafts CSS according to the [Turing CSS style guide](https://github.com/turingschool-examples/css)
+- Crafts markup according to the [Turing HTML style guide](https://github.com/turingschool-examples/html)
+- Application utilizes consistant naming for HTML classes and IDs, and follows suggested conventions.
+- Application uses an appropriate amount of [HTML semantic elements](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure). If `<div>` elements are used, they are only for styling purposes.
+- Application utilizes organizational conventions for the whole CSS stylesheet. This may look like - grouping native elements, typography styles, layout styles, etc. together.
+- Application utilizes data attributes to store information in the HTML
+
+✨WOW✨ can look like:
+- Developers use [BEM](http://getbem.com/), [SMACCS](http://smacss.com/), or another set of naming conventions for classes. _Note: If going this route, document it in your README and let your project manager know which naming convention will be used._
+- Application fully implements HTML that is accessible for individuals with visual disabilities. _Note: This will be checked using the Chrome Extenstion [WAVE](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US) audit tool, and 0 errors & 0 contrast errors are expected._
+</section>
+
+<section class="note">
+### Note about using the WAVE tool
+
+To get this extension to work with local files, follow these steps:
+- Install
+- Right click the WAVE extension
+- Click "Manage Extensions"
+- Flip the "Allow access to file URLs" switch
+- Success!
+</section>
+
+<section class="answer">
```

Comment:
Thank you for updating the rubric to match the *Not Yet*, *Yes*, or *WOW* structure!